x86-64_darwin fails for some reason

```
./txr   tests/018/noclose.txr  > tst/tests/018/noclose.out ; case $? in ( 0 ) diff -u tests/018/noclose.expected tst/tests/018/noclose.out ;; ( 13 ) printf "SKIP %s\n" tests/018/noclose.txr ; exit 0 ;; ( * ) exit 1 ;; esac
touch tst/tests/018/noclose.ok
touch tests/010/cons.expected
mkdir -p tst/tests/010/
./txr --gc-debug  tests/010/cons.tl  > tst/tests/010/cons.out
if ! diff -u tests/010/cons.expected tst/tests/010/cons.out ; then exit 1 ; fi
touch tst/tests/010/cons.ok
touch tests/010/hash.expected
mkdir -p tst/tests/010/
./txr --gc-debug  tests/010/hash.tl  > tst/tests/010/hash.out
if ! diff -u tests/010/hash.expected tst/tests/010/hash.out ; then exit 1 ; fi
touch tst/tests/010/hash.ok
touch tests/010/json.expected
mkdir -p tst/tests/010/
./txr --gc-debug  tests/010/json.tl  > tst/tests/010/json.out
if ! diff -u tests/010/json.expected tst/tests/010/json.out ; then exit 1 ; fi
touch tst/tests/010/json.ok
touch tests/010/qquote.expected
mkdir -p tst/tests/010/
./txr --gc-debug  tests/010/qquote.tl  > tst/tests/010/qquote.out
if ! diff -u tests/010/qquote.expected tst/tests/010/qquote.out ; then exit 1 ; fi
touch tst/tests/010/qquote.ok
touch tests/010/range.expected
mkdir -p tst/tests/010/
./txr --gc-debug  tests/010/range.tl  > tst/tests/010/range.out
./txr: unhandled exception of type type-error:
./txr: from: #<garbage: 7faa281c5ee0> is not of type range
./txr: during evaluation at tests/010/range.tl:74 of form (equal #:g0248 #:g0249)
make: *** [Makefile:472: tst/tests/010/range.ok] Error 1
rm tests/010/qquote.expected tests/010/span-var.expected tests/008/call-2.expected tests/010/json.expected tests/010/cons.expected tests/008/mdo.expected tests/010/hash.expected tests/008/no-stdin-hang.expected tests/010/range.expected
error: builder for '/nix/store/3klpgdnywrk4avy5c8nh5s09xwlkgj2v-txr-289.drv' failed with exit code 2;
```


maybe just disable tests or mark it broken.

--- a/pkgs/tools/text/txr/default.nix
+++ b/pkgs/tools/text/txr/default.nix
@@ -69,5 +69,6 @@ stdenv.mkDerivation (finalAttrs: {
     license = lib.licenses.bsd2;
     maintainers = with lib.maintainers; [ AndersonTorres dtzWill ];
     platforms = lib.platforms.all;
+    broken = stdenv.isDarwin && stdenv.isx86_64; # ofborg fails while testing
   };
 })

